ui/StatusQ/src/StatusQ/Popups/StatusMenuItemDelegate.qml

```diff
@@ -46,50 +91,21 @@ MenuItem {
             bold: false
             italic: false
         }
-    }
-
-    Component {
-        id: indicatorIcon
-
-        StatusIcon {
-            width: d.assetSettings.width
-            height: d.assetSettings.height
-            rotation: d.assetSettings.rotation
-            icon: d.assetSettings.name
-            color: {
-                const c = d.assetSettings.color;
-                if (!Qt.colorEqual(c, "transparent"))
-                    return c;
-                if (!root.enabled)
-                    return Theme.palette.baseColor1;
-                if (d.isStatusDangerAction)
-                    return Theme.palette.dangerColor1;
-                return Theme.palette.primaryColor1;
-            }
-        }
-    }
 
-    Component {
-        id: indicatorLetterIdenticon
-
-        StatusLetterIdenticon {
-            width: d.assetSettings.width
-            height: d.assetSettings.height
-            color: d.assetSettings.bgColor
-            name: root.text
-            letterSize: 11
+        readonly property StatusIdenticonRingSettings defaultRingSettings: StatusIdenticonRingSettings {
+            ringPxSize: Math.max(1.5, d.assetSettings.width / 24.0)
```

Comment:
Not sure if worth it, but we seem to be doing the same thing here: https://github.com/status-im/status-desktop/pull/8436/files#diff-a295c8811d526e0c83e97c3c9431b689b5cce6e9ff2b0c73bcc39b003a7a18edR31 maybe it makes sense to have a little utils function for that so we ensure this doesn't get out of sync when changes are made?